bindings/src/client/entities/Object.js

```diff
@@ -246,6 +246,10 @@ export class _NetworkObject extends _Object {
             if (shouldCancel()) return;
 
             this.#handle = natives.createObject(model, this.alt.pos.x, this.alt.pos.y, this.alt.pos.z, false, false, false);
+            if (!this.#handle) {
+                console.error('Failed to create object using native:', model);
+                return;
+            }
```

Comment:
it might be better to throw an error, so its handled in `catch`
```suggestion
            if (!this.#handle) {
                throw new Error('Failed to create object using native:', model);
            }
```